### PR TITLE
quick script to test plan dropdown

### PR DESCRIPTION
**Pull request by TrentonWalkup**

### CODE REVIEW

**Comment by Gregorma** [comment]
LGTM

**Comment by taylorlacy** [comment]
LGTM

File: demo/PROD_regressionSuite/corp_statedropdown.js [hunk 1]
```diff
@@ -2,18 +2,75 @@ var i = require('../tools.js') //required to import tools
...
-  click('.purchase-path-link') //clicking the join now button
-  browser.waitUntil(function () {
-    return browser.isVisible('body > div.ng-scope > div > div.loader-holder > div > div') === false}, 10000)
-  browser.pause(5000)
-  assertText('.sections__plans__region', i.user.stateFull) //asserting that the correct state passes into checkout
+  browser.pause(3000)
+  if (browser.isVisible('.price-panel') != true) {
+    if(browser.isVisible('div.ls-c-plan-pricing-title:nth-child(1) > div:nth-child(1)') != true){
+      browser.click(state)
+    }
```
Comment (roppel5ls) [inline demo/PROD_regressionSuite/corp_statedropdown.js]: No going into checkout?

**Comment by TrentonWalkup** [comment]
added a viewport re-sizing. We need to get to a point where we figure out functionality vs userpath testing. Where does that line blur, where do we go. We also need a tools file for running things locally.

**Comment by robertvazquezlegalshield** [comment]
LGTM

**Pull request merged**